corporate/tests/test_stripe.py

```diff
@@ -1277,6 +1280,12 @@ def test_attach_discount_to_realm(self, *mocks: Mock) -> None:
         self.assertEqual([4000 * self.seat_count],
                          [item.amount for item in invoice.lines])
 
+        # Check the logging
```

Comment:
No need of this comment. It's obvious what the code does. Also remove the blank line above.